gui-apps/swaylock-effects/swaylock-effects-1.6.10.ebuild

```diff
@@ -8,12 +8,14 @@ inherit bash-completion-r1 meson
 DESCRIPTION="Screen locker for Wayland"
 HOMEPAGE="https://github.com/jirutka/swaylock-effects"
 
+MY_PV=$(ver_rs 2 '-')
 if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/jirutka/swaylock-effects.git"
 else
 	SRC_URI="https://github.com/jirutka/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64"
+	S="${WORKDIR}"/${PN}-${MY_PV}
+	KEYWORDS=""
```

Comment:
Why were these removed?

Reply:
There are two version of swaylock-effects exist in here, one form [[mortie](https://github.com/mortie/swaylock-effects)](url)(swaylock-effect-1.6.4) and the other from [[jirutka](https://github.com/jirutka/swaylock-effects)](url)(swaylock-effects-1.6.10).
The swaylock-effects-1.6.10 is from [[jirutka](https://github.com/jirutka/swaylock-effects)](url) doesn’t work with sway 1.8 but due to the package name, portage will think this is the newer version and pull it when update.

Reply:
Then lets remove the old one entirely.